src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpClassification.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpClassification : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public CSharpClassification(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(CSharpClassification))
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public void Verify_Color_Of_Some_Tokens()
```

Comment:
I don't think anybody else has been doing the underscores-between-words thing, it looks like just this and WinForms.